x-pack/plugins/observability/server/services/slo/delete_slo.test.ts

```diff
@@ -29,7 +29,7 @@ describe('DeleteSLO', () => {
 
   describe('happy path', () => {
     it('removes the transform, the roll up data and the SLO from the repository', async () => {
-      const slo = createSLO(createAPMTransactionErrorRateIndicator());
+      const slo = createSLO();
```

Comment:
💬 The createSLO fixture function uses an APMTransactionErrorRateIndicator by default.